packages/volto/src/components/manage/Blocks/Block/EditBlockWrapper.jsx

```diff
@@ -105,15 +114,58 @@ const EditBlockWrapper = (props) => {
         <div className={`ui drag block inner ${type}`}>
           {children}
           {selected && !required && editable && (
-            <Button
-              icon
-              basic
-              onClick={() => onDeleteBlock(block, true)}
-              className="delete-button"
-              aria-label={intl.formatMessage(messages.delete)}
+            <div
+              style={{
+                position: 'absolute',
+                top: '-3px',
+                right: '-35px',
+                display: 'flex',
+                zIndex: 10,
+              }}
```

Comment:
move styles into css please, and give a class name to this wrapper. No use inline styles please